charts/crowdsec/README.md

```diff
@@ -75,6 +75,118 @@ lapi:
           key: dbPassword
 ```
 
+## Setup for AppSec (WAF) with Traefik bouncer as example
+
+Below a basic configuration for AppSec (WAF)
+
+```
+# your-values.yaml (option 1)
+appsec:
+  enabled: true
+  acquisitions:
+    - source: appsec
+      listen_addr: "0.0.0.0:7422"
+      path: /
+      appsec_config: crowdsecurity/virtual-patching
+      labels:
+        type: appsec
+  env:
+    - name: COLLECTIONS
+      value: "crowdsecurity/appsec-virtual-patching"
+
+# This allows the LAPI pod to register and communicate with the appsec pod
```

Comment:
Add to readme they should not change `${REGISTRATION_TOKEN}` as it will break things